backend/btrixcloud/crawls.py

```diff
@@ -135,7 +139,6 @@ async def list_crawls(
                     "as": "crawlConfig",
```

Comment:
Should we also remove this lookup, as it was only being used for the name, and looks like no longer used!

Reply:
Good call, now removed!